frontend/pages/bounty-programs/[bountyProgramSlug]/index.tsx

```diff
@@ -60,28 +61,60 @@ const BountyPrograms: NextPage = () => {
         padding="15px"
         borderRadius="4px"
       >
-        <Heading fontSize="lg" fontFamily="mono">
-          🕵️‍♀️ {bountyProgram.name} Bounty Question Proposals
-        </Heading>
-        <Text fontFamily="mono" fontSize="sm">
-          <br />A space to propose and gather questions relevant to{" "}
-          {bountyProgram.name}. What do you want to know about{" "}
-          {bountyProgram.name}? Questions defined, and ranked here will form the
-          basis of future Analytics Bounties by MetricsDAO!
-          <br />
-          <br />
-          See a question you want answered? Upvote it! 
-        </Text>
-        <Button
-          marginTop="15px"
-          variant="secondary"
-          onClick={() => {
-            mixpanel.track("click:bounty_question_tips");
-            onOpen();
-          }}
-        >
-          🤔 How to Write a Good Question (click me to learn) 🤔
-        </Button>
+        {bountyProgram.customTitle == undefined ? (
+          <Heading fontSize="lg" fontFamily="mono">
+            🕵️‍♀️ {bountyProgram.name} Bounty Question Proposals
+          </Heading>
+        ) : (
+          <Heading fontSize="lg" fontFamily="mono">
+            🕵️‍♀️ {bountyProgram.customTitle}
+          </Heading>
+        )}
+        {bountyProgram.customDescription == undefined ? (
+          <Text fontFamily="mono" fontSize="sm">
+            <br />A space to propose and gather questions relevant to{" "}
+            {bountyProgram.name}. What do you want to know about{" "}
+            {bountyProgram.name}? Questions defined, and ranked here will form
+            the basis of future Analytics Bounties by MetricsDAO!
+            <br />
+            <br />
+            See a question you want answered? Upvote it!
+          </Text>
+        ) : (
+          <Text fontFamily="mono" fontSize="sm">
+            <br />
+            {bountyProgram.customDescription}
```

Comment:
Can you please check if we can insert <br> in the `bountyProgram.customDescription` ? As you see in the text above if we have to configure through JSON then we need ability to add line breaks one way or other.